subsys/bluetooth/controller/ll_sw/ll_adv.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct ll_adv_set {
+	u8_t  chl_map:3;
+	u8_t  filter_policy:2;
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_PRIVACY)
+	u8_t      rl_idx : 5;
+	u8_t      own_addr_type : 2;
+	u8_t      id_addr_type : 1;
+	u8_t      id_addr[BDADDR_SIZE];
```

Comment:
Btw, when using bitfields we need the members to be defined both in big and little endian, which apparently is not done here, actually many parts of the controller code seems to be doing the same.